app_levels/en.json

```diff
@@ -3,11 +3,11 @@
     "applevel_1":  "Installable",
     "applevel_2":  "Installable in all situations",
     "applevel_3":  "Can be updated",
-    "applevel_4":  "Single Sign On support",
+    "applevel_4":  "Backup and restore support",
     "applevel_5":  "Clean",
-    "applevel_6":  "Backup and restore support",
+    "applevel_6":  "Available for the community",
     "applevel_7":  "Successfully pass functional tests",
-    "applevel_8":  "Respect main guidelines",
-    "applevel_9":  "Respect all guidelines",
+    "applevel_8":  "High quality app",
+    "applevel_9":  "Respect highter guidelines",
```

Comment:
"higher" or "highest"... ?

Reply:
"higher"
I'm sure we will find more stuff to do in the future.

Reply:
```suggestion
    "applevel_9":  "Respect higher guidelines",
```
:wink: